hummingbot/client/command/gateway_command.py

```diff
@@ -156,23 +110,27 @@ async def _create_gateway(self):
                                   app_warning_msg=str(e))
             return
         self._notify("Creating new Gateway docker container...")
-        container_id = self._docker_client.create_container(image = f"{docker_repo}:gateway-v2",
-                                                            name = gateway_container_name,
-                                                            ports = [5000],
-                                                            volumes=[gateway_conf_path, certificate_path, log_path],
-                                                            host_config=self._docker_client.create_host_config(
-                                                                port_bindings={5000: 5000},
-                                                                binds={gateway_conf_path: {'bind': '/usr/src/app/conf/',
-                                                                                           'mode': 'rw'},
-                                                                       certificate_path: {'bind': '/usr/src/app/certs/',
-                                                                                          'mode': 'rw'},
-                                                                       log_path: {'bind': '/usr/src/app/logs/',
-                                                                                  'mode': 'rw'}}))
+        host_config = await self.docker_ipc(("create_host_config",
+                                             {"port_bindings": {5000: 5000},
+                                              "binds": {"gateway_conf_path": {"bind": "/usr/src/app/conf/",
```

Comment:
the `gateway_conf_path` and similar keys in the dictionary should be without quotes.